CHANGELOG.md

```diff
@@ -1,5 +1,66 @@
 # Database Tools Changelog
 
+## 100.2.1
+
+_Released 2020-11-13
+
+We are pleased to announce version 100.2.1 of the MongoDB Database Tools.
+
+This release bumps the Go driver version to 1.4.2, adds a releases archive for the Tools, and logs when issuing count commands in `mongodump` and `mongoexport` so slow collection counts aren't misinterpreted as hangs. It also fixes a bug that prevented mongorestore from scaling properly.
```

Comment:
```suggestion
This release includes a `mongorestore` performance improvement, a fix for a bug affecting highly parallel `mongorestore`s, and an observability improvement to `mongodump` and `mongoexport`, in addition to a number of internal build/release changes.
```
I think we should just stick to a high-level summary of the user-facing issues here -- anyone who needs to know details can go to the tickets!